nservicebus/upgrades/5to6.md

```diff
@@ -196,4 +196,17 @@ NServiceBus version 6 is no longer scanning nested directories for assemblies. Y
 
 ### Include moved to Exclude
 
-In version 6 the API has been changes to a "Exclude a list" approach. See [Assemblies to scan](/nservicebus/hosting/assembly-scanning.md#assemblies-to-scan) for more information.
\ No newline at end of file
+In version 6 the API has been changes to a "Exclude a list" approach. See [Assemblies to scan](/nservicebus/hosting/assembly-scanning.md#assemblies-to-scan) for more information.
+
+
+## RepeatedFailuresOverTimeCircuitBreaker has been made internal
+
+If your are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
+
+## Transport seam
+
+`IDispatchMessages` have been obsoleted and is replaced by `IPushMessages`. The interfaces are basically equivalent so if you're implementing your own transport you should be able to just implement the new interface.
+
+### Corrupted messages
+
+The core will now pass the error queue address to the transport via the `IDispatchMessages.Init(PushSettings)` API. The transport is expected to move the corrupted message to the specified error queue when detected.
```

Comment:
I'd say yes, as soon as the seam is stable we should document it? (@distantcam is updating RabbitMQ as we speak)